src/ai/ai.controller.ts

```
@@ -0,0 +1,80 @@
+import { Body, Controller, Param, Post, Query } from '@nestjs/common';
```

ESLint

[error] 'Query' is defined but never used.

(@typescript-eslint/no-unused-vars)